.github/workflows/bootstrap_script.yml

```diff
@@ -69,11 +69,17 @@ jobs:
 
     - name: install OS & Python packages
       run: |
-        # disable apt update, we don't really need it,
-        # and it does more harm than good (it's fairly expensive, and it results in flaky test runs)
-        # sudo apt update
         # for modules tool
-        sudo apt install lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev
+        APT_PKGS="lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev"
+
+        # Avoid apt-get update, as we don't really need it,
+        # and it does more harm than good (it's fairly expensive, and it results in flaky test runs)
```

Comment:
@Flamefire Can you also extend the comment to explain why we do try to `apt-get update` if `apt-get install` failed, since it seems to go against what the current comment states...

Reply:
I had already changed that comment to "avoid" but agree to be a bit more explicit why we then do it